### DISCUSSION TITLE
Is it possible to add a function button for uploading images to the sidebar of the page menu?

### DISCUSSION THREAD

**Discussion by ghost** [goharbor/harbor#19667] [Ideas]
The command line or interface is of course possible, but it would be better if Harbor supports it natively.
![image](https://github.com/goharbor/harbor/assets/71524776/32f61906-a373-4d06-9e01-67581458074d)

**Comment by zyyw** [comment]
No, only way is docker push

**Reply by ghost** [reply]
In enterprises, the demand for uploading through the UI interface is still quite strong, especially for developers or business personnel who are not familiar with Docker.
